### DISCUSSION TITLE
New block returning id

### DISCUSSION THREAD

**Discussion by Soarezin** [PiranhaCMS/piranha.core#1945] [Q&A]
So, i'm trying to make a custom block for a website, but when i place it on the page it returns this
"
Id
5fc99dd2-a23c-4660-b080-7eaf4100a039
Type
TeatroFeevaleCMS.Models.Blocks.SolicitacaoPautaBlock
"

anyone has ever passed through this?

**Comment by tidyui** [comment]
Adding a custom block only makes it available in the content model and the manager interface. To render it in your web application you need to add a DisplayTemplate for it. These are normally located in the `DisplayTemplates` folder in your `Views` or `Pages` folder and should have the same name as your block class, in your case `SolicitacaoPautaBlock.cshtml`.

**Reply by Soarezin** [reply]
Hey man, i've done this and it still returning the block id, later i'll post here the code. I compared my custom block with others custom blocks, also that i've made and it's basically the same, still cant figure out why this is happening
